wasm/js/transform_avro/README.md

```diff
@@ -27,44 +41,68 @@ npm test       # run mocha tests
 npm run build  # bundle js application with webpack 
 ```
 
-Create Redpanda topic:
+## Create the topic
+
+Create the topic where the producer app will send messages.
+This step is required since the consumed topic must exist prior to deploying the wasm coprocessor.
+
+```bash
+> rpk topic create market_activity --brokers localhost:19092
+TOPIC            STATUS
+market_activity  OK
+```
 
+List all current topics:
 ```bash
-rpk topic create market_activity --brokers localhost:19092
-rpk topic list --brokers localhost:19092
+> rpk topic list --brokers localhost:19092
+NAME             PARTITIONS  REPLICAS
+_schemas         1           1
+market_activity  1           1
 ```
 
-Deploy the transform script to Redpanda:
+The `_schemas` topic is automatically generated when the schemas are uploaded to the registry.
+
+## Deploy the transform script to Redpanda
 
 ```bash
-rpk wasm deploy dist/main.js --name json2avro --description "Transforms JSON to AVRO" --brokers localhost:19092
+> rpk wasm deploy dist/main.js --name json2avro --description "Transforms JSON to AVRO" --brokers localhost:19092
```

Comment:
nit: Maybe for the docs keeping the kafka port as `9092` would lend to less possibilities of confusion